Source/Resources/Content/AadIam/ExternalUserDirectorySettings/Configuration.json

```diff
@@ -1,5 +1,6 @@
 {
   "allowInvitations": true,
+  "description":"What is it: Defines external collaborator settings including who can invite external collaborators and if restrictions will be placed on external users. \n Simeon Notes:  Defaults in Azure allow all users and guests to invite external parties, Simeon's baseline only allows authorized users with the guest inviter role to invite external parties. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/external-identities/delegate-invitations",
```

Comment:
**What does this do?**
Defines external collaborator settings including who can invite external collaborators and if restrictions will be placed on external users. The baseline only allows authorized users with the 'Guest inviter' role to invite external parties.

**Why should you use this?**
Defaults in Azure allow all users and guests to invite external parties, so the baseline configuration is far more secure. 

**What is the end user impact?**
Users will not be able to invite external collaborators unless authorized.

**Learn more**
[Configure B2B external collaboration settings](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/delegate-invitations)